playground/signalr/serverless/SignalRServerless.Functions/Function.cs

```diff
@@ -0,0 +1,169 @@
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace SignalRServerless.Functions;
+
+// Reference: https://github.com/aspnet/AzureSignalR-samples/tree/main/samples/DotnetIsolated-BidirectionChat
+public class Functions
+{
+    private readonly ILogger _logger;
+
+    public Functions(ILoggerFactory loggerFactory)
+    {
+        _logger = loggerFactory.CreateLogger<Functions>();
+    }
+
+    [Function("index")]
+    public HttpResponseData GetWebPage([HttpTrigger(AuthorizationLevel.Anonymous)] HttpRequestData req)
```

Comment:
Instead of `HttpResponseData`/`HttpRequestData`, let's update he sample to use the new `HttpRequest`/`IResult` pattern from the Microsoft.Azure.Functions.Http.AspNetcore package instead. Moving the sample to the existing Functions end-to-end should help with this.